packages/api/src/hooks/useRemoteConfig.ts

```diff
@@ -0,0 +1,23 @@
+import { useQuery } from '@tanstack/react-query';
+import initData from '../remote_config.json';
+
+const remoteConfigQuery = async function () {
+    if (!process.env.REMOTE_CONFIG_URL) {
```

Comment:
is this browser executed? I would think process.env is only build-time variable, not runtime/browser object? how does it work?

Reply:
I believe this var will be baked into the JS code. Isn't it?

Reply:
I think webpack will replace this

Reply:
since i've added it to webpack config it would be accessible on runtime as well

Reply:
makes sense, pretty confusing though to bring clearly backend variables/objects/concepts (process object) to frontend :)